cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,17 +23,121 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
-`Release 3.5.0 <http://docs.cask.co/cdap/3.5.0/index.html>`__
+`Release 3.5.1 <http://docs.cask.co/cdap/3.5.1/index.html>`__
 =============================================================
 
 Known Issues
 ------------
+
+- :cask-issue:`CDAP-7175` - Though an underlying issue has been fixed in the upgrade tool
```

Comment:
this seems very verbose. Perhaps just refer to the doc on upgrade, instead of including it here again?

Also, does starting with "Though" seem to suggest the user is expected to have some prior knowledge about this?